src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -176,6 +181,11 @@ private function getListOfProjectType(): ?array {
     return $array;
   }
 
+  public function getCurlCommand(string $projectAccessToken, string $hostPath, int $projectId, string $ciPath): string {
```

Comment:
You've already got the gitlab PHP SDK and glab CLI in the Code Studio commands, does this really need to be a curl command?

Reply:
using glab cli changes were not reflecting: https://acquia.atlassian.net/browse/GL-2368?focusedCommentId=8264317